gordon.toml.example

```diff
@@ -12,6 +12,10 @@ publish = "cleanup"
 [core.logging]
 level = "info"
 handlers = ["syslog"]
+format = "%(created)f %(levelno)d %(message)s"
+date_format = "%Y-%m-%dT%H:%M:%S"
+address = ["10.99.0.1", "514"]
```

Comment:
I'm not sure it's a problem (because [SysLogHandler doesn't specifically mention it will fail with a string port](https://docs.python.org/3/library/logging.handlers.html#logging.handlers.SysLogHandler), but it looks like ulogger [won't cast the second value to an int](https://github.com/spotify/ulogger/blob/master/ulogger/syslog.py#L81).

Reply:
dang, my eyes went to [this](https://github.com/spotify/ulogger/blob/master/ulogger/syslog.py#L94) thinking it did. hm I wonder if there's a graceful way to handle this in gordon. Trying to avoid fixing ulogger but I think that's probably what I should do...

Reply:
@descent3me mind taking a look at this? it's just a quick+dirty fix: https://github.com/spotify/ulogger/pull/1